rudder_airflow_provider/hooks/rudderstack.py

```diff
@@ -33,16 +33,17 @@ def trigger_sync(self):
             extra_options={"check_response": False})
         if resp.status_code in (200, 204, 201):
             logging.info('Job triggered for sourceId: %s', self.source_id)
+            return resp.json().get('runId')
         elif resp.status_code == 409:
```

Comment:
Not raising any exception if there is a sync already running and the return value will be `None`

This PR simply ignores the `wait_for_completion` flag and task succeeds if a new run didn't start.

@AchuthaSourabhC @RanjeetMishra thoughts on this?

Reply:
On another note, not many customers are using and they do not upgrade so frequently as well. We can discuss and plan to raise exception for this scenario and update docs for new version.
We have to share new version with Kape and doc can let them know how to integrate.

Reply:
While i don't know what would be impact of changing for customer. Ignoring also seems bit dangerous.

Reply:
@AchuthaSourabhC @RanjeetMishra Let's keep the behaviour same and not stop the execution if there is no new job run triggered. In this case, `wait_for_completion` does not wait for any ongoing job to finish.